projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -296,6 +296,11 @@
 	max-width: 25%;
 }
 
+.wp-block-jetpack-contact-form .grunion-field-width-33-wrap {
+	flex: 1 1 calc( 33.33% - calc(var(--wp--style--block-gap, 1.5rem) * 1) );
+	max-width: 33.33%;
```

Comment:
We could stick to `33%` instead of `33.33%` for consistency between how input reads vs CSS value, but this CSS value gives more accurate layout and "33%" looks way better in the control, so I think this is fine.

Reply:
Alternative approach:

<img width="277" alt="Screenshot 2025-05-09 at 12 36 24" src="https://github.com/user-attachments/assets/d688f2ce-5f7b-4b11-9251-ec6f8c746e6d" />